tests/src/JIT/opt/Devirtualization/sealeddefault.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+public class Base
+{
+    public virtual int Foo() { return 33; }
+    
+    static BaseSealed s_Default = new BaseSealed();
+
+    public static Base Default => s_Default;
+}
+
+sealed class BaseSealed : Base {}
+
+// The jit ought to be able to devirtualize the call to b.Foo below.
```

Comment:
Will update;  `Base.Default.Foo()` will devirtualize, but not, as you note, `x.Foo()` currently won't.